src/reconcile/rec_visibility.c

```diff
@@ -1067,6 +1064,129 @@ __rec_upd_select(WT_SESSION_IMPL *session, WTI_RECONCILE *r, WT_CELL_UNPACK_KV *
     return (0);
 }
 
+/*
+ * __rec_upd_select_inmem --
+ *     Select the update to write to disk image for in-memory btree. For in-memory btree, we select
+ *     the first globally visible update in the update chain that is not a tombstone. If there is no
+ *     stable update found, write the oldest committed update.
+ *
+ * @param session The session handle. @param r Reconciliation state structure. @param vpack Unpacked
```

Comment:
No need for this?

Reply:
Tried to add comments for @param but s_all keeps putting all of them together on the same line, so I ended up just remove these to bypass s_all